src/libcore/str/pattern.rs

```diff
@@ -1178,8 +1178,8 @@ impl TwoWaySearcher {
 trait TwoWayStrategy {
     type Output;
     fn use_early_reject() -> bool;
-    fn rejecting(usize, usize) -> Self::Output;
-    fn matching(usize, usize) -> Self::Output;
+    fn rejecting(a: usize, b: usize) -> Self::Output;
```

Comment:
I wonder if there are more descriptive names we could use here? I don't know enough about this piece of code, but @Kimundi might.